README.md

```diff
@@ -27,3 +49,5 @@ Watch mode implies that when any part of the code is modified, the modified test
 Running the `npm build` command builds the app for production.
 It correctly bundles React in production mode and optimizes the build for the best performance.
 [Read more](https://facebook.github.io/create-react-app/docs/deployment)
+
+##### Spring Boot (?)
```

Comment:
You could add this description for Spring boot:
Spring Boot is an open source Java-based framework used to create a micro Service. It is developed by Pivotal Team and is used to build stand-alone and production ready spring applications.